HISTORY.rst

```diff
@@ -3,6 +3,15 @@ Release History
 
 Unreleased Changes
 ------------------
+* Adding custom exception ``ReclassificationMissingValuesError`` that can 
+  occur when mapping raster values to dictionary keys, where a raster value 
+  is not represented in the dictionary as a key. The custom exception takes 
+  3 arguments that are set as attributes:
+  * ``msg`` - a string for the error message
+  * ``missing_values`` - a list of raster values not found in the dictionary
+  * ``value_map`` - the dictionary the raster values were attempted to map to
+  This error is now raised in ``reclassify_raster`` and can be caught 
+  to better debug the mismatch when using a table to reclassify a raster. 
 * Correcting the docstring for ``pygeoprocessing.numpy_array_to_raster`` to
```

Comment:
I've got a proposed re-word of this section below. The ``msg`` is part of the base exception so I don't think that needs mentioning and if you're up for my suggestion below I think you can drop ``value_map``. 

`Added a custom exception ``ReclassificationMissingValuesError`` to ``pygeoprocessing.reclassify_raster`` that raises when a raster pixel value is not represented in ``value_map``. This custom exception provides a list of missing raster pixel values in a ``missing_values`` attribute that allows the caller access to the pixel values that are missing through a Python type rather than indirectly through an error message.`

Reply:
Yea, I think this is better stated then what I have. I'll update it.

Reply:
Updated statement here: https://github.com/natcap/pygeoprocessing/pull/106/commits/d0bcb5436d271f8f1c58a6d44853cdbd782fb185